### DISCUSSION TITLE
Do we need algorithm: auto ?

### DISCUSSION THREAD

**Discussion by karaka200** [symfony/symfony#45143] [Q&A] [answered]
Given the existence of such a line in `security.yaml` :

```
    password_hashers:
        Symfony\Component\Security\Core\User\PasswordAuthenticatedUserInterface: 'auto'
```

Is there a need to use these lines?

```
    password_hashers:
        Symfony\Component\Security\Core\User\PasswordAuthenticatedUserInterface: 'auto'
+       App\Entity\User:
+           algorithm: auto
```

**Comment by derrabus** [comment, marked as answer]
If your user already implements that interface (which it should), you don't need those lines.

**Reply by karaka200** [reply]
These two lines exist by default

```
password_hashers:
    Symfony\Component\Security\Core\User\PasswordAuthenticatedUserInterface: 'auto'
```

So we do not need these two extra lines that make with `maker bundle`. It is true?

```
App\Entity\User:
    algorithm: auto
```

**Reply by derrabus** [reply]
Maker Bundle probably creates them to be on the safe side. You could've modified that setting or deleted it.

If `App\Entity\User` is your only user object, you can either delete the generic default setting or the one generated by MakerBundle.
